mantidimaging/core/operations/loader.py

```diff
@@ -20,8 +20,9 @@ def _find_operation_modules() -> list[BaseFilterClass]:
         if not ispkg:
             continue
 
-        if getattr(sys, 'frozen', False):
+        if getattr(sys, 'frozen', False) and "test.support" not in module_name:
             # If we're running a PyInstaller executable then we need to use a full module path
+            # Need to prevent importing standard library test modules found by pkgutil
             module_name = f'mantidimaging.core.operations.{module_name}'
```

Comment:
Should be jumping to the next iteration here, instead of adjusting the module_name?

```python
        if getattr(sys, 'frozen', False):
            # Need to prevent importing standard library test modules found by pkgutil
            if "test.support" in module_name:
                 continue
            # If we're running a PyInstaller executable then we need to use a full module path
            module_name = f'mantidimaging.core.operations.{module_name}'
```

Reply:
Ah yes true! Interestingly, just not adjusting the `module_name` worked

Reply:
Might have changed if from failing on the non-existant `mantidimaging.core.operations.test.support` to getting the standard library `test.support`